src/domain/app/layout/PageWrapper.tsx

```diff
@@ -50,8 +52,8 @@ const PageWrapper: React.FC<Props> = ({
       <Helmet>
         <html lang={locale} />
         <title>{translatedTitle}</title>
+        {description && <meta name="description" content={t(description)} />}
```

Comment:
Perhaps `meta.default.description` could be used as default, when description is not given as argument

Reply:
fixed